tests/CppUTestExt/IEEE754PluginTest_c.c

```diff
@@ -25,6 +25,10 @@
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include <CppUTest/CppUTestConfig.h>
+
+#ifdef CPPUTEST_HAVE_FENV
```

Comment:
```c++
void set_overflow_c(void)
{
    f = 1e38f;
    f *= f;
}
```
appears to work, at least on my machine. If it survives Travis, then I guess I could use that.